internal/runners/packages/import.go

```diff
@@ -143,7 +148,24 @@ func (i *Import) Run(params *ImportRunParams) (rerr error) {
 	// Solve the runtime.
 	rtCommit, err := bp.FetchCommit(stagedCommitId, proj.Owner(), proj.Name(), nil)
 	if err != nil {
-		return errs.Wrap(err, "Failed to fetch build result for previous commit")
+		var buildplannerErr *response.BuildPlannerError
+		if errors.As(err, &buildplannerErr) {
+			// When importing CycloneDX and SPDX SBOMs, we put all packages in the 'private/<org>'
+			// namespace, which will fail to solve. That is expected. However, we still want to update the
+			// local commit so the user can see what was imported, even if the runtime is not viable.
+			for _, verr := range buildplannerErr.ValidationErrors {
+				if strings.Contains(verr, "non-existent namespace: private/") {
+					if err := localcommit.Set(proj.Dir(), stagedCommitId.String()); err != nil {
+						return locale.WrapError(err, "err_package_update_commit_id")
+					}
+					return errImportSbomSolve
```

Comment:
This is too specific. We should not have awareness of sboms on the state tool side, and definitely should not be string matching errors from the API to localize them here. 

I think we should just make this the mechanic for import; we always update the commit regardless of whether the builder errored out. And we forward the API error, which should already be helpful enough on its own.